app/src/main/java/app/revanced/integrations/patches/components/ShortsFilter.java

```diff
@@ -58,7 +58,9 @@ public ShortsFilter() {
                 "shorts_shelf",
                 "inline_shorts",
                 "shorts_grid",
-                "shorts_video_cell"
+                "shorts_video_cell",
+                "shorts_pivot_item",
+                "rich_grid_row"
```

Comment:
I've not seen similar UI anywhere on YouTube apart from these shorts. Though hiding that grid can cause YouTube to fetch new ones and strain the API. In addition to that, hiding shorts in that tab is useless. If you don't want to see shorts, don't go to that tab. Previous issues should be discussed

Reply:
It was in `shorts_video` path.

    (ID: null, Buffer-size: 5624): 
    rich_grid_row.eml|cfc05328964830eb|CellType|ContainerType|shorts_video_item.eml|5ed154c144eb7bf8|ContainerType|

Reply:
We can also use `shorts_video` but it has to be `pathFilterGroup` instead of `identifierFilterGroup`. It will get rid of `shorts_video_cell`, `shorts_pivot_item`, and `rich_grid_row`.